powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ServiceController.java

```diff
@@ -41,16 +41,19 @@ public class ServiceController {
     private static final Logger logger = LoggerFactory.getLogger(ServiceController.class);
 
     private final ApplicationConfiguration applicationConfiguration;
-    private final BuildProperties buildProperties;
+    private BuildProperties buildProperties;
 
     /**
      * Service constructor.
      * @param applicationConfiguration PowerAuth admin configuration.
-     * @param buildProperties Build info.
      */
     @Autowired
-    public ServiceController(ApplicationConfiguration applicationConfiguration, BuildProperties buildProperties) {
+    public ServiceController(ApplicationConfiguration applicationConfiguration) {
```

Comment:
We already use the pattern with Setter injection elsewhere. Let's keep it here for now... I changed this just because the Tomcat deploy was crashing on Idea build, it should never happen with standard Maven builds.